package.json

```diff
@@ -42,79 +42,81 @@
     "label tool"
   ],
   "devDependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-react": "^7.16.7",
+    "@babel/cli": "^7.23.0",
+    "@babel/core": "^7.23.3",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
```

Comment:
![image](https://github.com/epam/deps-fe-labeling-tool/assets/35468938/a9b6002f-7d52-4d01-925b-1a070fa8c5bd)

so would be nice to replace it to @babel/plugin-transform-class-properties